lasagne/layers/cuda_convnet.py

```diff
@@ -99,7 +99,9 @@ class Conv2DCCLayer(CCLayer):
         An initializer for the weights of the layer. This should initialize the
         layer weights to a 4D array with shape
         ``(num_filters, num_input_channels, filter_rows, filter_columns)``.
+
          If automatic dimshuffling is disabled (see notes), the shape should be
+
         ``(num_input_channels, input_rows, input_columns, num_filters)``
```

Comment:
+1 for removing Sphinx warnings! But what exactly does it say? Is this the only/correct way to fix it? What do you mean by "long `...`"?